assets/_scss/core/_grid.scss

```diff
@@ -32,4 +32,8 @@
       @include omega();
     }
   }
+}
+
+.grid-box {
```

Comment:
Since so much in the library is already using this class, we should resolve this in another PR.